Source/Tests/LinuxSyscalls/Syscalls/Info.cpp

```diff
@@ -19,6 +19,23 @@
 
 namespace FEX::HLE {
   void RegisterInfo() {
+    REGISTER_SYSCALL_IMPL(uname, [](FEXCore::Core::InternalThreadState *Thread, struct utsname *buf) -> uint64_t {
+      struct utsname Local{};
+      if (::uname(&Local) == 0) {
+        strncpy(buf->nodename, Local.nodename, sizeof(Local.nodename));
+        static_assert(sizeof(Local.nodename) <= 65);
+      }
+      else {
+        strcpy(buf->nodename, "FEXCore");
+      }
+      strcpy(buf->sysname, "Linux");
+      strcpy(buf->release, "5.0.0");
+      strcpy(buf->version, "#" FEXCORE_VERSION);
+      // Tell the guest that we are a 64bit kernel
+      strcpy(buf->machine, "x86_64");
```

Comment:
Should this really be "x86_64" for x32 syscalls?

Reply:
Yes. This is the exact behaviour of calling uname with a 32bit executable.
it's the machine architecture of the kernel, not the program itself.